src/fable-library/Util.ts

```diff
@@ -61,16 +61,6 @@ export interface IEqualityComparer<T> {
   GetHashCode(x: T): number;
 }
 
-export interface ICollection<T> {
-  readonly Count: number;
```

Comment:
@alfonsogarciacaro Did that interface move somewhere else?

Reply:
No, I just noticed it's not used anywhere so I removed it. If you need it we can put it back though right now all calls to `ICollection` members are replaced with something else so I don't know if there's any type implementing it.

Unfortunately I realized it's difficult to use mangled interfaces in Typescript because of the overload hash (we could try though) so I had to [add these as exceptions for the mangling](https://github.com/fable-compiler/Fable/blob/596502bbe7b7c4d56dd81980bee88006e87ddaa9/src/Fable.Transforms/FSharp2Fable.Util.fs#L1038-L1049) and tried to keep them tracked.

Thanks a lot for your comments @ncave! BTW, could you please also take a look at `Fable2Babel`? Not sure how the changes to the compilation of attached members affect the compilation with classes. At least I guess we need to check if the interface is mangled or not here: https://github.com/fable-compiler/Fable/blob/596502bbe7b7c4d56dd81980bee88006e87ddaa9/src/Fable.Transforms/Fable2Babel.fs#L1808-L1810

Reply:
@alfonsogarciacaro It's being used [here](https://github.com/fable-compiler/Fable/blob/master/src/Fable.Transforms/Fable2Babel.fs#L1747).